cookbooks/swift/recipes/setup.rb

```diff
@@ -221,6 +221,12 @@
     :base_uri => node['base_uri'],
   })
 end
+template "/home/#{node['username']}/.aws/credentials" do
```

Comment:
Since there isn't actually any templating, we could use `cookbook_file` and stick it in the `files/` tree, but w/e.